commcare_connect/opportunity/models.py

```diff
@@ -400,6 +401,8 @@ class Payment(models.Model):
     )
     confirmed = models.BooleanField(default=False)
     confirmation_date = models.DateTimeField(null=True)
+    # This is used to indicate Payments made to Network Manager organizations
+    organization = models.ForeignKey(Organization, on_delete=models.DO_NOTHING, null=True, blank=True)
```

Comment:
I think this can just be a boolean like `org_payment` and it would be great to have a constraint to make sure  opportnity access is always null when it is True

Reply:
Also is this set in this PR or a different one. I couldn't find it.

Reply:
this is set in this PR. https://github.com/dimagi/commcare-connect/pull/387